lib/rmt/cli/products.rb

```diff
@@ -35,21 +38,70 @@ def list
   end
   map ls: :list
 
-  desc 'enable', 'Enable mirroring of product repositories by product ID or product string.'
+  desc 'enable TARGETS', 'Enable mirroring of product repositories by a list of product IDs or product strings.'
   option :all_modules, type: :boolean, desc: 'Enables all free modules for a product'
-  def enable(target)
-    change_product(target, true, options[:all_modules])
+  def enable(*targets)
+    change_products(targets, true, options[:all_modules])
   end
 
-  desc 'disable', 'Disable mirroring of product repositories by product ID or product string.'
-  def disable(target)
-    change_product(target, false, false)
+  desc 'disable TARGETS', 'Disable mirroring of product repositories by a list of product IDs or product strings.'
+  def disable(*targets)
+    change_products(targets, false, false)
   end
 
   protected
 
+  def change_products(targets, set_enabled, all_modules)
+    success = true
+    targets = clean_target_input(targets)
+    raise RMT::CLI::Error.new('No product ids supplied') if targets.empty?
+
+    targets.each do |target|
+      change_product(target, set_enabled, all_modules)
+    rescue ProductNotFoundException => e
+      puts e.message
+      success = false
+    end
+
+    raise RMT::CLI::Error.new("Not all products were #{set_enabled ? 'enabled' : 'disabled'}.") unless success
+  end
+
   def change_product(target, set_enabled, all_modules)
+    # This will return multiple products if 'SLES/15' was used
+    products = find_products(target)
+    raise ProductNotFoundException.new("No product found for target '#{target}'.") if products.empty?
+    puts "Found product(s) by target #{target}: #{products.map(&:friendly_name).join(', ')}."
+
+    products.each do |product|
+      puts "For #{product.friendly_name}:"
+
+      product_with_extensions = [product]
+      if set_enabled
+        extensions = all_modules ? Product.free_and_recommended_modules(product.id).to_a : Product.recommended_extensions(product.id).to_a
+        unless extensions.empty?
+          puts 'Enabling additional extensions:'.indent(2)
+          extensions.each { |extension| puts extension.name.indent(4) }
+          product_with_extensions.push(*extensions)
+        end
+      end
+
+      puts "#{set_enabled ? 'Enabling' : 'Disabling'} repositories:".indent(2)
```

Comment:
The text....:

> Found product(s) by target X:
> For PRODUCT:
> Enabling additional extensions:
> Enabling repositories:

I don't know, it just doesn't read smoothly for me.

Also, since repositories belong to specific products, maybe we shouldn't separate both lists, but put the repositories below the product they belong to? Like:

```
    Basesystem Module
      Enabled repositories:
      SLE-Module-Basesystem15-SP1-Pool
      SLE-Module-Basesystem15-SP1-Updates

```

Anwyay, I suggested to @tmuntaner that we could show this to the doc team, they could have better/different ideas about this. These really are just my personal, super biased opinions.